pom.xml

```diff
@@ -9,7 +9,7 @@
   </parent>
 
   <artifactId>idm-user-management</artifactId>
-  <version>0.2.3-SNAPSHOT</version>
+  <version>0.2.3</version>
```

Comment:
-1 leave it on a SNAPSHOT build. the version numbering for release builds is automatically handled by the `mvn release:prepare` and `mvn release:perform` commands